src/main/java/pl/poznan/put/jsontools/logic/JsonTransform.java

```diff
@@ -0,0 +1,7 @@
+package pl.poznan.put.jsontools.logic;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public interface JsonTransform {
+    JsonNode execute(JsonNode json);
```

Comment:
There should be one interface in the Decorator design pattern. 
https://refactoring.guru/design-patterns/decorator